File: Models/Enums/OrderStatus.cs
using System.ComponentModel.DataAnnotations;

namespace ShopWebApp.Models.Enums
{
    public enum OrderStatus
    {
        [Display(Name = "Pending")]
        Pending,
        [Display(Name = "Shipped")]
        Shipped,
        [Display(Name = "Finished")]
        Finished,
        [Display(Name = "Cancelled")]
        Cancelled
    }
}

File: ClientApp/src/utilities/orderStatus.js
const setOrderStatus = (id) => {
    switch (id) {
        case 0:
            return "Pending";
        case 1:
            return "Shipped";
        case 2:
            return "Finished";
        case 3:
            return "Cancelled";
    }
};

export default setOrderStatus;
